Example/ViewController.swift

```diff
@@ -81,6 +81,15 @@ final class ViewController: UIViewController {
     @IBAction func stop(_ sender: UIButton) {
         player.stop()
     }
+    
+    @IBAction func metadata(_ sender: Any) {
+        let newMetadata = ModernAVPlayerMetadata(title: "Updated title",
```

Comment:
can you add like short timestamp in one of the parameter just to display new fresh data when tap button

Reply:
Good idea